providers/src/airflow/providers/http/provider.yaml

```diff
@@ -25,6 +25,7 @@ state: ready
 source-date-epoch: 1734534857
 # note that those versions are maintained by release manager - do not update them manually
 versions:
+  - 5.0.1
```

Comment:
I think it should be 5.1.0 - there aare far too many changes to justify just "patchlevel"